java-checks-testkit/src/main/java/org/sonar/java/checks/verifier/internal/JavaCheckVerifier.java

```diff
@@ -160,7 +160,7 @@ private static void addIssues(JavaFileScannerContextForTests scannerContext, Mul
       var quickfixes = scannerContext.getQuickFixes().get(textSpan);
       if (quickfixes != null) {
         for (var qf : quickfixes) {
-          verifierIssue = verifierIssue.addQuickFix(convertQuickFix(qf));
+          verifierIssue = (MultiFileVerifier.Issue) verifierIssue.addQuickFix(convertQuickFix(qf));
```

Comment:
I can create a PR on sonar-analyzer-commons for this, and a ticket on sonar-java side? Since if we don't release we cannot make this change right now here. How does that sound?